runtime/lib/ttnn/operations/eltwise/tertiary/where.cpp

```diff
@@ -0,0 +1,25 @@
+// SPDX-FileCopyrightText: (c) 2024 Tenstorrent AI ULC
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "where.h"
+#include "tt/runtime/detail/ttnn.h"
+#include "tt/runtime/ttnn/operations/utils.h"
+
+namespace tt::runtime::ttnn::operations::where {
+
+void run(const ::tt::target::ttnn::WhereOp *op, ProgramContext &context) {
+  ProgramTensorPool &tensorPool = context.tensorPool;
+  runWhereOp(op, tensorPool, ::ttnn::where);
```

Comment:
Seems like you forgot to remove this line, not sure how is it compiling in CI O.o